_sections/live-share-code-with-me-support-vscode.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Visual Studio Code
+tab: vscode
+---
+
+#### Starting a Live Share Session 
+
+To start a Live Share session, you should select the Live Share tab on the Share Code popup modal, and select the project you would like to share. 
+
+![Start Live Share](/uploads/codeshare-start-liveshare.jpg "Start Live Share"){:class="screenshot"}
+
+Once you have selected a project you will be prompted to switch back to your IDE.
+
+![Live Share Open Repo](/uploads/liveshare-open-repo.jpeg "Live Share Open Repo"){:class="screenshot"}
+
+If you are using Live Share and you haven't signed in yet, you will be prompted to sign in. 
+
+![Live Share Sign In](/uploads/codeshare-liveshare-sign-in.jpeg "Live Share Sign In"){:class="screenshot"}
+
+
+ Once you have signed in, you are good to go!
+
+#### Sharing with GitLive
+
+Sharing with the built-in codeshare and then realise you need the extra power of Live Share? No problem, you can switch from the built-in codeshare to Live Share at any time in VS Code.
+
+![Codeshare Livesharing](/uploads/codeshare-livesharing.jpeg "Codeshare Livesharing"){:class="screenshot"}
```

Comment:
this doesnt belong here, its out of context. also rename the title to Switiching from GitLive code share to Live Share

Reply:
put it at the bottom of the section